packages/lesmis/src/components/MapOverlaysPanel/index.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ *
+ */
+export * from './MapOverlaysPanel';
+export * from './MapOverlaysPanelContainer';
```

Comment:
The container is used in another place later for the map section which is why it is it's own component